### ISSUE TITLE
[Feature Request] option to block scroll

### ISSUE THREAD

**Issue by monkeydri** [VikLiegostaiev/react-page-scroller#18]
Would be nice to have an option to disallow scroll, ex : a multi-step website where each step is a page in `<ReactPageScroller />` and step should be completed before moving on next one.

**Comment by VikLiegostaiev** [comment]
Hi, @monkeydri , thanks for pull request. I will update npm version soon.

**Comment by VikLiegostaiev** [comment]
@monkeydri 
published: https://github.com/VikLiegostaiev/react-page-scroller/tree/1.4.3

**Comment by monkeydri** [comment]
thanks a lot ! another simple PR is in the pipeline
